module-services/service-desktop/ServiceDesktop.cpp

```diff
@@ -241,6 +245,54 @@ sys::ReturnCodes ServiceDesktop::InitHandler()
         return btMsgHandler->handle(msgl);
     });
 
+    connect(typeid(db::NotificationMessage), [&](sys::Message *msg) {
+        if (!notificationsClearTimer.isActive()) {
```

Comment:
please at least create a function, for this code :) 
1. will you add different notifications handling in the service desktop? 
2. will new if's be added here?
3. can we create a class from NotificationsEntries instead of working everywhere on a generic vector?

Reply:
Moved to function in a separate class.
Will be reworked in the next PR's.